workspaces/arborist/lib/add-rm-pkg-deps.js

```diff
@@ -125,7 +125,7 @@ const deleteSubKey = (pkg, depType, name, replacedBy) => {
 }
 
 const rm = (pkg, rm) => {
-  for (const depType of new Set(saveTypeMap.values())) {
+  for (const depType of saveTypeMap.values()) {
```

Comment:
saveTypeMap.values has `peerDependencies` twice, this will need to be reverted or rethought.